tollway/__main__.py

```diff
@@ -1,31 +1,41 @@
 import time
-import random
-from typing import Optional
 
 import typer
 from faker import Faker
 from faker_vehicle import VehicleProvider
+from typing_extensions import Annotated
 
-from tollway.vehicle import get_tollways, create_tollway, create_vehicle, create_payload
+from tollway.constants import (
+    ALL_EVENTS_COUNT,
+    DATE_VARIATION_RATE,
+    INCLUDE_DUPLICATE_RATE,
+    INCLUDE_LATE_RATE,
+    Help,
+)
+from tollway.events import process_duplicate_event, process_late_event
 from tollway.utils import get_date_variation, write_to_file
-from tollway.constants import DATE_VARIATION_RATE, INCLUDE_LATE_RATE, INCLUDE_DUPLICATE_RATE, ALL_EVENTS_COUNT
-from tollway.events import process_late_event, process_duplicate_event
-
+from tollway.vehicle import create_payload, create_tollway, create_vehicle, get_tollways
 
 tollways = get_tollways()
 fake = Faker()
 fake.add_provider(VehicleProvider)
 
 
 def main(
-        total_events: Optional[int] = typer.Option(default=1),
-        event_rate: Optional[float] = typer.Option(default=1.0),
-        output_file: Optional[bool] = typer.Option(default=False),
-        output_filename: Optional[str] = typer.Option(default="tollway-traffic.json"),
-        date_variation: Optional[bool] = typer.Option(default=False),
-        include_late: Optional[bool] = typer.Option(default=False),
-        include_duplicate: Optional[bool] = typer.Option(default=False),
-        ):
+    total_events: Annotated[int, typer.Option(help=Help.TOTAL_EVENTS.value)] = 1,
+    event_rate: Annotated[float, typer.Option(help=Help.EVENT_RATE.value)] = 1.0,
```

Comment:
have not used Help enum, nice thing is that you have all the strings in one place now!

Reply:
while they're somewhat related, main thought was it would be nice to have all strings in one place